packages/arb-util/arbos/arbos.go

```diff
@@ -16,6 +16,8 @@ var ARB_BLS_ADDRESS = ethcommon.HexToAddress("0x00000000000000000000000000000000
 var ARB_FUNCTION_TABLE_ADDRESS = ethcommon.HexToAddress("0x0000000000000000000000000000000000000068")
 var ARB_OWNER_ADDRESS = ethcommon.HexToAddress("0x000000000000000000000000000000000000006B")
 
+var ARB_NODE_INTERFACE_ADDRESS = ethcommon.HexToAddress("0x000000000000000000000000000000000000006E")
```

Comment:
I'd put this at a much higher address to differentiate it and not conflict with future arbos "contracts"

Reply:
Agree with this, but we'll need to charge ArbOS. I'll flag @edfelten on this question

Reply:
This was changed to 200 to keep it near the arbos address block, but leave plenty of room for future expansion